app/plugins/adp-backend/src/service/deliveryProgrammeRouter.ts

```diff
@@ -53,37 +52,63 @@ export async function createProgrammeRouter(
   });
 
   router.get('/deliveryProgramme', async (_req, res) => {
-    const data = await deliveryProgrammesStore.getAll();
-    res.json(data);
+    try {
+      const data = await deliveryProgrammesStore.getAll();
+      res.json(data);
+    } catch (error) {
+      const errMsg = (error as Error).message;
```

Comment:
Same as above, loggign the error object would be better than just its message property I feel